ApplyToBecomeInternal/ApplyToBecome.Data/Services/AcademyConversionProjectRepository.cs

```diff
@@ -1,128 +1,139 @@
-				searchModel.DeliveryOfficerQueryString = deliveryOfficerFilter;
-			}
-
-			if (statusFilters != null)
-			{
-				IEnumerable<string> projectedStatuses = statusFilters.SelectMany(x =>
-					_invertedAliasedStatuses.ContainsKey(x.ToLowerInvariant())
-						? new[] { x, _invertedAliasedStatuses[x.ToLowerInvariant()] }
-						: new[] { x });
-
-				searchModel.StatusQueryString = projectedStatuses.ToArray();
-			}
-
-			if (regionsFilter != default && regionsFilter.Any())
-			{
-            HttpResponseMessage regionResponse = await _apiClient.GetSelectedRegionsAsync(regionsFilter);
-
-            List<int> regionUrnResponse = await regionResponse.Content.ReadFromJsonAsync<List<int>>();
-            searchModel.RegionUrnsQueryString = regionUrnResponse;
-         }
-		}
-
-		public async Task<ApiResponse<AcademyConversionProject>> GetProjectById(int id)
-		{
-         HttpResponseMessage response = await _apiClient.GetProjectByIdAsync(id);
-			if (!response.IsSuccessStatusCode)
-			{
-				return new ApiResponse<AcademyConversionProject>(response.StatusCode, null);
-			}
-
-			AcademyConversionProject project = await response.Content.ReadFromJsonAsync<AcademyConversionProject>();
-			return new ApiResponse<AcademyConversionProject>(response.StatusCode, project);
-		}
-
-		public async Task<ApiResponse<AcademyConversionProject>> UpdateProject(int id, UpdateAcademyConversionProject updateProject)
-		{
-         HttpResponseMessage response = await _apiClient.UpdateProjectAsync(id, updateProject);
-			if (!response.IsSuccessStatusCode)
-			{
-				return new ApiResponse<AcademyConversionProject>(response.StatusCode, null);
-			}
-
-			AcademyConversionProject project = await response.Content.ReadFromJsonAsync<AcademyConversionProject>();
-			return new ApiResponse<AcademyConversionProject>(response.StatusCode, project);
-		}
-
-		public async Task<ApiResponse<ProjectFilterParameters>> GetFilterParameters()
-		{
-			HttpResponseMessage response = await _apiClient.GetFilterParametersAsync();
-
-			if (response.IsSuccessStatusCode is false)
-				return new ApiResponse<ProjectFilterParameters>(response.StatusCode, null);
-
-			var filterParameters = await response.Content.ReadFromJsonAsync<ProjectFilterParameters>();
-
-			filterParameters.Statuses =
-				filterParameters.Statuses.Select(x => _aliasedStatuses.ContainsKey(x.ToLowerInvariant()) ? _aliasedStatuses[x.ToLowerInvariant()] : x)
-					.Distinct()
-					.OrderBy(x => x)
-					.ToList();
-
-			filterParameters.Regions = new List<string>
-			{
-				"East Midlands",
-				"East of England",
-				"London",
-				"North East",
-				"North West",
-				"South East",
-				"South West",
-				"West Midlands",
-				"Yorkshire and the Humber"
-			};
-			return new ApiResponse<ProjectFilterParameters>(response.StatusCode, filterParameters);
-		}
-	}
+   private readonly IReadOnlyDictionary<string, string> _aliasedStatuses = new Dictionary<string, string> { { "converter pre-ao (c)", "Pre advisory board" } };
```

Comment:
I'm not sure what has changed here. Please could you point to the actual change.